tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -978,12 +980,12 @@ def teardown(self):
         self.remote_queue = None
         self.recording.pause()
 
-    def ensure_runner_stopped(self):
-        self.logger.debug("ensure_runner_stopped")
+    def _ensure_runner_stopped(self):
         if self.test_runner_proc is None:
```

Comment:
I see we only reset `test_runner_proc` to None when tearing down `TestRunnerManager`. I think this should be done when test runner is stopped.

Reply:
Done (and added a comment where `test_runner_proc.is_alive()` is used)